<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lithophane</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Image</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1256</width>
           <height>457</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="lImage">
            <property name="text">
             <string>Load a 1200x400 px image!</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Layer height:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="sbLayerHeight">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="prefix">
        <string/>
       </property>
       <property name="suffix">
        <string> mm</string>
       </property>
       <property name="minimum">
        <double>0.100000000000000</double>
       </property>
       <property name="maximum">
        <double>0.300000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
       <property name="value">
        <double>0.300000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Build plate dimensions</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,2,0,2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Width (X):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sbBedWidth">
        <property name="minimum">
         <number>160</number>
        </property>
        <property name="maximum">
         <number>350</number>
        </property>
        <property name="value">
         <number>250</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Depth (Y):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sbBedDepth">
        <property name="minimum">
         <number>160</number>
        </property>
        <property name="maximum">
         <number>350</number>
        </property>
        <property name="value">
         <number>200</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="bLoad">
     <property name="text">
      <string>Browse &amp;&amp; Load image</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="bGenerate">
     <property name="text">
      <string>Generate &amp;&amp; Save GCODE</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="pbTotal">
     <property name="value">
      <number>0</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
